### ISSUE TITLE
404 Error when listing packages on the webserver

### ISSUE THREAD

**Issue by hubby2004** [NuGet/NuGetGallery#3451]
Hi,

I followed your steps for creating a NuGet server in IIS. However, when I click on the Packages link, it throws a 404 error instead of listing the packages I have uploaded. Did I miss something?

Thanks!
Mike

**Comment by scottbommarito** [comment]
Are you trying to run the [Gallery](https://github.com/NuGet/NuGetGallery) or the [Server](https://github.com/NuGet/NuGet.Server)?

If you are running the Gallery, does every endpoint besides https://nuget.localtest.me/packages work correctly, or do other pages 404 as well?

If you are running the Server, what is the url you are trying to hit? I don't know a whole lot about it but my understanding was that it didn't have a packages display page but I could be wrong.

**Comment by hubby2004** [comment]
I am trying to run the gallery. When I run https://nuget.localtest.me/packages, it throw a 404 error (only for packages webpage). I can upload packages, register users, etc. without any issues. I can browse directly to the package page (e.g. https://nuget.localtest.me/packages/winrar/5.40.0) but not https://nuget.localtest.me/packages

**Comment by hubby2004** [comment]
I rebuilt the NuGet Gallery following the steps [here](https://github.com/NuGet/NuGetGallery#build-and-run-the-gallery-in-arbitrary-number-easy-steps) and still have the problem. But now, I have a much "better informing" error stating the following: The file extension for the requested URL does not have a handler configured to process the request on the Web server.
I am not sure what handler am I missing. I installed HttpPlatformHandler v1.2 via web platform but no luck.

**Comment by hubby2004** [comment]
I found out what the issue was (stupid mistake). The Lucene module was being overwritten when I was publishing the project. So I published the project in a different location as the build and now it works. Thanks for the assistance!

**Comment by sakshamsaxena** [comment]
@hubby2004 Hey, could you elaborate this a bit ? I am stuck on the exact issue. (I just started this yesterday)

**Comment by sakshamsaxena** [comment]
@hubby2004 indeed that's what I ended up doing. Ran away from the bewitched IIS which was causing this behaviour and deployed to Azure. Works excellent there. It was a configuration issue most definitely as we even went ahead to run the Gallery on Debug mode via VS 2017 on the server itself, which ran rather smoothly, but the deployed one was simply a rascal, unwilling to give any meaningful clue even after turning off the custom errors.
